ulid_test.go

```diff
@@ -621,7 +621,8 @@ func TestMonotonicSafe(t *testing.T) {
 	t.Parallel()
 
 	var (
-		safe = ulid.DefaultEntropy()
+		rng  = rand.New(rand.NewSource(time.Now().UnixNano()))
+		safe = &ulid.LockedMonotonicReader{MonotonicReader: ulid.Monotonic(rng, 0)}
```

Comment:
Re-use of default entropy among parallel tests was creating test flakes when the per-ms entropy ceiling was hit.

Reply:
Help me understand please. What kind of flakes? The default entropy should be safe. If it's not, that's a problem?

Reply:
It's definitely safe. When the tests were running with a high level of concurrency, it was possible to make so many Reads from the DefaultEntropy in a single millisecond (from different tests) that it no longer provided monotonicity.

Reply:
Strictly speaking, the documentation for [Monotonic](https://pkg.go.dev/github.com/oklog/ulid/v2#Monotonic) is incorrect when it says

> Monotonic returns an entropy source **that is guaranteed to yield strictly increasing entropy bytes for the same ULID timestamp**. On conflicts, the previous ULID entropy is incremented with a random number between 1 and `inc` (inclusive).

Rather, the entropy will be strictly increasing **until** adding that random number goes beyond the available entropy space. That's the limit we hit.

Reply:
OK, LGTM